app/models/course.rb

```diff
@@ -0,0 +1,5 @@
+class Course < ApplicationRecord
```

Comment:
I noticed that you changed the name of the model from "class" to "course" as you guys mentioned in the WhatsApp group and the application is working fine for me at the moment, however, I noticed that the name of the controller is still "ClassesController" and the route for it is still "classes"
it's up to you if you want to change those to courses or not
other than that I don't see any problem with it.
great job guys!

Reply:
Thanks for your review @MahdiAghaali . You are right and I an going to change this ASAP.